src/via.ts

```diff
@@ -7,22 +10,59 @@ type ProjectConfig = {
   services: Record<string, ServiceConfig>
 }
 
+async function initConfigDirectory(projectName: string) {
+  await ensureDir(`${viaRoot}/projects`)
+  const projectConfigFilePath = `${viaRoot}/projects/${projectName}.yaml`
+  if (await fileExists(projectConfigFilePath)) {
+    console.log(`configuration file for project "${projectName}" already exists`)
+    return
+  }
+
+  const exampleProjectConfigContent = `services:
+  database:
+    path: ${Deno.env.get("HOME")}/path/to/service
+    actions:
+      start: docker-compose up -d
+      stop: docker-compose down
+`
+    await Deno.writeTextFile(projectConfigFilePath, exampleProjectConfigContent)
+
+  console.log(`configuration file for project was created at "${projectConfigFilePath}"`)
+  console.log('Edit this config for your needs')
+}
+
 const viaRoot = Deno.env.get("HOME") + '/.via'
 
 async function getConfig() {
   const config: Record<string, ProjectConfig> = {}
   const decoder = new TextDecoder("utf-8");
+  const viaProjectConfigPath = viaRoot + '/projects';
+  if (!await fileExists(viaProjectConfigPath)) {
+    console.error('The config directory does not exists')
+    console.error('Create your first project configuration by running "v init <project name>"')
+    Deno.exit(1)
+  }
   for await (const dirEntry of Deno.readDir(viaRoot + '/projects')) {
-    if(!dirEntry.name.endsWith('.json')) {
+    const extension = dirEntry.name.split('.').pop()
+    if(!dirEntry.isFile || !['yaml', 'yml'].includes(extension)) {
```

Comment:
```suggestion
    if(!dirEntry.isFile || !extension || !['yaml', 'yml'].includes(extension)) {
```
TS complains otherwise